### PR TITLE
Support app install and uninstall

### PR DESCRIPTION
**Pull request by aluedeke**
- app install through multipart form upload
- uninstall using the bundleid

### CODE REVIEW

File: restapi/api/app_endpoints.go [hunk 1]
```diff
@@ -138,3 +142,95 @@ func KillApp(c *gin.Context) {
...
+	}
+
+	if file.Size > 200*1024*1024 { // 100 MB limit
+		c.JSON(http.StatusRequestEntityTooLarge, GenericResponse{Error: "file size exceeds the 200MB limit"})
+		return
+	}
+
+	appDownloadFolder := os.Getenv("APP_DOWNLOAD_FOLDER")
+	if appDownloadFolder == "" {
+		appDownloadFolder = "/tmp"
```
Comment (danielpaulus) [inline restapi/api/app_endpoints.go:178]: please use os.TempDir otherwise windows won't work

File: restapi/api/app_endpoints.go [hunk 2]
```diff
@@ -138,3 +142,95 @@ func KillApp(c *gin.Context) {
...
+	}
+
+	dst := appDownloadFolder + "/" + uuid.New().String() + ".ipa"
```
Comment (danielpaulus) [inline restapi/api/app_endpoints.go:181]: please use path.Join, same reason as above

**Review by danielpaulus — CHANGES_REQUESTED** [review]
some minor changes for our windows friends

**Review by danielpaulus — APPROVED** [review]
nice!

File: restapi/api/app_endpoints.go (hunk 1 above)
Comment (aluedeke) [reply restapi/api/app_endpoints.go]: fixed

File: restapi/api/app_endpoints.go (hunk 2 above)
Comment (aluedeke) [reply restapi/api/app_endpoints.go]: fixed

**Pull request merged**